libs/platform/experience/src/services/layout/plugins/styles/canvas/canvas-style.model.ts

```diff
@@ -6,6 +6,13 @@ declare global {
      */
     background?: string;
 
+    /**
+     * Sets the overflow behavior of an element's content. If an element's content is larger than
```

Comment:
Does it really make sense to explain css properties? In any case we are not providing possible values and not affecting default implementation in the browser. Maybe we can just give a link to MDN or W3?

Reply:
We had it on other elements and hence I stayed consistent with it. The longer term vision is that we'd have docs on those attributes for both devs and users in EB. Inline docs is not working for that, I know.

Reply:
lets keep it for now